media/js/firefox/hello/index.js

```diff
@@ -160,12 +160,40 @@
         } else {
             // if Fx is version 34 or lower (no Hello support) display update messaging in footer
             $('#ctacopy-oldfx').show();
+
+            $('.download-link').on('click', function(e) {
+                var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
```

Comment:
Code wise this look fine :+1: 

As the two function body's are pretty similar in each callback however, could it be worth combining the two and calling a single function that just get's passed the label name as a param?